src/components/CollectionsAndPalettes/common/CollectionsHeader/CollectionsHeader.module.scss

```diff
@@ -0,0 +1,76 @@
+@import "styles/global";
+@import "styles/variables.scss";
+
+.Header {
+  background-color: black;
+  display: flex;
+  flex-direction: row;
+  color: white;
+  padding: $margin-small-4 $margin-huge-5;
+  column-gap: $margin-medium-3;
+}
+.headerLogo {
+  h1 {
+    font-size: 26px;
+    color: white;
+  }
+  div {
+    display: flex;
+    align-items: center;
+    margin-right: 0 !important;
+    svg {
+      height: 40px;
+
+      path {
+        fill: white;
+      }
+    }
+  }
+}
+.navLinks {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  column-gap: $margin-medium;
+
+  .navLink {
+    display: flex;
+    flex-direction: row;
+    align-items: baseline;
```

Comment:
I think we can use `align-items: center` here, in case the window is a bit narrower and the text wraps.